README.md

```diff
@@ -172,27 +188,79 @@ command.
 There are three views for reviewing a pull request on GitHub: **Conversation**,
 **Commits**, and **Files changed**.
 
-#### Conversation ####
+**Conversation**
+
+- Comments on the pull request show up here, including any inline comments that
+  are added when viewing the file diffs. A summary of submitted commits is also
+  listed.
+- Use this section to submit general feedback about the pull request.
+
+**Commits**
+
+- A more detailed list of commits included in the pull request, that will allow
+  a reviewer to view each individually.
+- Usually before merging, you'll want all the commits from that change to be
+  squashed into a single commit. This so that the commit can show the whole
+  change that was made, and so that there aren't a bunch of typo fixes in the
+  repository's commit history!
+
+**Files changed**
 
-Comments on the pull request show up here, including any inline comments that
-are added when viewing the file diffs. A summary of submitted commits is also
-listed.
+- The `diff` view of the pull request. Lets the reviewer see the full patch in
+  its entirety, and add inline comments on changes.
+- Use this section to submit line-specific feedback.
 
-Use this section to submit general feedback about the pull request.
+#### Giving feedback ####
 
-#### Commits ####
+Everyone is invited to give feedback regardless of experience. Comments can
+touch on:
 
-A more detailed list of commits included in the pull request, that will allow a
-reviewer to view each individually.
+- phrasing of any text added
+- tags on event posts (do they make sense? are they existing tags?)
+- lines that are over 80 characters long (which we avoid)
```

Comment:
"which we avoid except for certain cases like the metadata at the top of event files" ?